Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -168,6 +170,14 @@
 #import <pal/cocoa/TranslationUIServicesSoftLink.h>
 #import <pal/mac/DataDetectorsSoftLink.h>
 
+#if ENABLE(WRITING_TOOLS_UI)
+namespace WebKit {
+void showSwapCharactersViewRelativeToRectOfView(NSRect positioningRect, NSView *positioningView);
```

Comment:
This should be in a header.  Also, it currently looks like they are unused, but I assume that's about to change with more upstreaming.

Reply:
For this upstreaming PR, I didn't want to refactor anything, so I'm just moving things 1:1, as I was advised. After finishing all upstreaming and renaming, I'm going to immediately refactor this and some other things